CONFIG.md

```diff
@@ -478,12 +479,52 @@ from the project directory when building a package. The above includes README
 files and files under bin/ in the project's package.
 
 **packages.dest.docroot**: Specify where within the package directory the `srcFiles`
-should be placed. Defaults to the package root. For Acquia set this to
-'/docroot'.
+should be placed. Defaults to the package root. For Acquia set this to '/docroot'.
 
 **packages.dest.devResources**: Specify where within the package directory the
 `projFiles` should be placed. Defaults to package root.
 
+
+### Project Operations
+
+Some of the most common use cases for any Drupal site are the basic operations
+of installation, running database updates, and clearing the cache. These are not
+difficult operations, but many developers appreciate the common shorthand a
+little grunt magic can provide.
+
+To keep this flexible the new Operations system offers nothing out of the box,
+but facilitates creating lightweight commands or callouts to the shell for more
+advanced scripts you want easily run on the project by the entire team.
+
+This should remind you of the **Themes Scripts** described above.
+
+```
+{
+  "scripts": {
+    "install": "drush <%= config.alias %> install <%= config.profile %> -yv"
+    "update": "drush <%= config.alias %> updatedb -yv && drush <%= config.alias %> features-revert-all -yv"
```

Comment:
Good point. The only downside is how soupy this starts to look:

```
<%= config.drush.cmd %> <%= config.alias %> install <%= config.profile %> -yv
```

It also seems like a likely error users will make in their own configurations.

What if we automatically replace `drush` in the configuration with our vendored version unless it's preceded by a !, such as `!drush`?